MANIFEST.in

```diff
@@ -1,5 +1,7 @@
 include LICENSE *.rst *.txt *.md
 graft wagtail
-prune wagtail/wagtailadmin/static_src
-global-exclude __pycache__
+prune **/__pycache__
+prune **/static_src
 global-exclude *.py[co]
+global-exclude .gitignore
+global-exclude wagtail-icomoon.json
```

Comment:
question: given wagtail-icomoon lives under `wagtail/admin/static_src/wagtailadmin/fonts` will it not be excluded via `prune **/statics_src`? or is it copied to static by the front-end tooling?

Reply:
Yes, it is currently being copied at `wagtail/admin/static/wagtailadmin/fonts/wagtail-icomoon.json`. I don’t know by what or why though 😅 .